src/cgame/rocket/rocket.cpp

```diff
@@ -701,22 +704,8 @@ static EngineCursor engineCursor;
 
 void Rocket_SetActiveContext( int catcher )
 {
-	switch ( catcher )
-	{
-		case KEYCATCH_UI:
-			engineCursor.Show( true );
-			drawMenu = true;
-			break;
-
-		default:
-			if ( !( catcher & KEYCATCH_CONSOLE ) )
-			{
-				engineCursor.Show( false );
-			}
-
-			drawMenu = false;
-			break;
-	}
+	drawMenu = !!( catcher & KEYCATCH_UI );
```

Comment:
If this is a cast to boolean, would not it be better to use something easier to read than `!!`? Maybe `( catcher & KEYCATCH_UI ) != 0`?

Reply:
I was paranoid about compiler warnings, but no compiler is actually warning about int to bool conversion so I just removed the `!!`.